config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+
+  root "home#index"
+
+  namespace :api do
+    namespace :v1 do
+      resources :greetings
```

Comment:
- [Optional] I don't think it is a good practice to expose unnecessary endpoints by using resourceful routes for this specific. You got only `index` and `show` actions in the controller. More over, you needed a controller with just one action

Reply:
Fixed